### PR TITLE
fix: #320 stop mentoring button

### PR DESCRIPTION
**Pull request by Howl404**

### CODE REVIEW

File: mw-webapp/src/logic/waysTable/MentoringWaysTable.tsx [hunk 1]
```diff
@@ -111,51 +108,37 @@ export const MentoringWaysTable = (props: MentoringWaysTableProps) => {
...
+                      /**
+                       * TODO: use modal instead of confirm task #305
+                       */
+
+                      /**
+                       * CallBack triggered on press ok
+                       */
+                      const onOk = () => stopMentoring(props.uuid, row.original);
+
+                      const isConfirmed = confirm(`Are you sure you want to stop mentoring "${row.original.name}"?`);
```
Comment (tritonJS826) [inline mw-webapp/src/logic/waysTable/MentoringWaysTable.tsx:134]: If is Confirm is not boolean - rename it (wrong 'is" prefix), if it is boolean - remove '!!' on the next line

File: mw-webapp/src/logic/waysTable/MentoringWaysTable.module.scss [hunk 2]
```diff
@@ -4,3 +4,9 @@
   margin-left: 5px;
   cursor: pointer;
 }
+
+.modalButtons {
```
Comment (tritonJS826) [inline mw-webapp/src/logic/waysTable/MentoringWaysTable.module.scss:8]: Why do you need this stones, if you don't have appropriate modal component?

File: mw-webapp/src/logic/waysTable/MentoringWaysTable.tsx [hunk 3]
```diff
@@ -111,51 +108,37 @@ export const MentoringWaysTable = (props: MentoringWaysTableProps) => {
...
-                    }
-                    isOpen={isStopMentoringModalOpen}
-                    handleClose={() =>
-                      setIsStopMentoringModalOpen(false)
-                    }
+
+                    onClick={() => {
+
+                      /**
+                       * TODO: use modal instead of confirm task #305
```
Comment (tritonJS826) [inline mw-webapp/src/logic/waysTable/MentoringWaysTable.tsx:126]: Put comment in the right place, remove multiline commet

**Review by tritonJS826 — APPROVED** [review]

**Review by dayanych — APPROVED** [review]

File: mw-webapp/src/logic/waysTable/MentoringWaysTable.module.scss (hunk 2 above)
Comment (Howl404) [reply mw-webapp/src/logic/waysTable/MentoringWaysTable.module.scss]: Removed

File: mw-webapp/src/logic/waysTable/MentoringWaysTable.tsx (hunk 1 above)
Comment (Howl404) [reply mw-webapp/src/logic/waysTable/MentoringWaysTable.tsx:131]: Removed !!

File: mw-webapp/src/logic/waysTable/MentoringWaysTable.tsx (hunk 3 above)
Comment (Howl404) [reply mw-webapp/src/logic/waysTable/MentoringWaysTable.tsx]: Placed next to confirm in one line

**Review by SergioAJS — APPROVED** [review]

**Pull request merged**